doc-Provisioning_Virtual_Machines_and_Hosts/topics/Catalogs_and_Services.adoc

```diff
@@ -364,4 +364,165 @@ include::common/catalog-order.adoc[]
+
+To have a look at the sample provisioning dialog for Red Hat Virtualization, go to menu:Automation[Automate > Customization > Provisioning Dialogs > VM Provision] and select *Sample RedHat VM Provisioning Dialog*. This is a textual representation of the dialog you will get when you provision a virtual machine or instance.
+
+
+.Create a Service Dialog
+
+Using a service dialog, you can determine which options the users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic RHEL 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
+
+[NOTE]
+====
+You cannot create a service without assigning it a service dialog, so even if you do not want to expose any details to users, you must create at least a simple service dialog with a *Submit* and *Cancel* button.
+====
+
+A service dialog contains three components:
+
+* One or more *Tabs*.
+* Inside the tabs, one or more *Boxes*.
+* Inside the boxes, one or more *Elements*. The *Elements* contain methods, like check boxes, drop-down lists or text fields, to fill in the options on the provisioning dialog.
+
+[IMPORTANT]
+====
+The names of the elements must correspond to the options used in the provisioning dialog.
+====
+
+The following procedure outlines how to create a service dialog for your sample service:
+
+. Navigate to menu:Automation[Automate > Customization].
+. Click the *Service Dialogs* accordion.
+. Click image:1847.png[](*Configuration*), and then image:1862.png[](*Add a new Dialog*).
+. Enter basic details under *Dialog Information*:
+.. Enter `Provision RHV VM` in *Label*, and enter a description for the dialog in *Description*. 
+.. Select the *Submit* and *Cancel* buttons.
+. Add a new tab to the dialog:
+.. Click image:1862.png[](*Add*), then image:1862.png[](*Add a New Tab to this Dialog*).
+.. Enter `General` in *Label*. 
+.. Optionally, enter a description for the tab in *Description*.
+. Add a new box to the tab:
+.. Click image:1862.png[](*Add*), then image:1862.png[](*Add a New Box to this Tab*).
+.. Enter `Service and VM Name` in *Label*.
+.. Optionally, enter a description for the box in *Description*.
+.. Add a new element for the service name. Elements are controls that accept input.
```

Comment:
Moving the sentence 'Elements are controls that accept input' up to the intial explanation of elements under the three components of service dialogs might provide better structure for the reader. Its a good and clear expanation that might be best up front.

Reply:
Good suggestion - done.